### ISSUE TITLE
[Pricing Table]: QA: After applying text spacing, text content in the page is getting overlapped.

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#9161]
**Engineering info:**  
- Innovation Team engineer: 
- Innovation Team JIRA Ticket: https://jsw.ibm.com/browse/ADCMS-2456
- DPO consulting engineer: N/A

-----

### Description

Page element:
Text content

Issue : After applying text spacing, text content in the page is getting overlapped.

Expected: All the elements in the page should be visible clearly after applying text spacing as well.

User Impact: If there is a loss of content or functionality When users override author specified text spacing to improve their reading experience, it affects users with low vision / user suffering from dyslexia.

### Component(s) impacted

Pricing Table

### Browser

Firefox

### Carbon for IBM.com version

v1.36.0-rc.0

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components-experimental

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by jkaeser** [comment]
Hi there, @Gopi916! Can you provide a screenshot that demonstrates the problem, and better yet, steps to reproduce the issue? What tool are you using to adjust text spacing? Thank you!

**Comment by Gopi916** [comment]
Hi @jkaeser Please find the issue screenshot below.
![image](https://user-images.githubusercontent.com/61313055/180992638-a2eb4f03-a14b-4167-8e67-4b6423ed72e1.png)
I used Text spacing bookmarklet from w3.org website to adjust text spacing. Please let me know if you need more details. Thanks!

**Comment by jkaeser** [comment]
> I used Text spacing bookmarklet from w3.org website to adjust text spacing. Please let me know if you need more details. Thanks!

Thanks for the information, @Gopi916! I wasn't able to find that bookmarklet, and other methods for adjusting the text spacing aren't reproducing what you're seeing:

1. A different bookmarklet (https://holistica11y.com/text-spacing-bookmarklet-for-accessibility-testing/): <img width="1044" alt="Screen Shot 2022-08-05 at 11 59 26 AM" src="https://user-images.githubusercontent.com/12532727/183115982-d0d77d40-8c5c-44bf-81b7-57741399c397.png">
2. Using the browser's dev tools: <img width="1044" alt="Screen Shot 2022-08-05 at 2 16 16 PM" src="https://user-images.githubusercontent.com/12532727/183137268-f7bb453d-db32-46a7-880c-f6da6a4a1376.png">

Can you send me a link to the bookmarklet you're using or even paste it in a code snippet here?

**Comment by jkaeser** [comment]
> Thanks for the information, @Gopi916! I wasn't able to find that bookmarklet, and other methods for adjusting the text spacing aren't reproducing what you're seeing:
> 
> 1. A different bookmarklet (https://holistica11y.com/text-spacing-bookmarklet-for-accessibility-testing/): <img alt="Screen Shot 2022-08-05 at 11 59 26 AM" width="1044" src="https://user-images.githubusercontent.com/12532727/183115982-d0d77d40-8c5c-44bf-81b7-57741399c397.png">
> 2. Using the browser's dev tools: <img alt="Screen Shot 2022-08-05 at 2 16 16 PM" width="1044" src="https://user-images.githubusercontent.com/12532727/183137268-f7bb453d-db32-46a7-880c-f6da6a4a1376.png">
> 
> Can you send me a link to the bookmarklet you're using or even paste it in a code snippet here?

Bump @Gopi916 🙂

**Comment by jkaeser** [comment]
> > Thanks for the information, @Gopi916! I wasn't able to find that bookmarklet, and other methods for adjusting the text spacing aren't reproducing what you're seeing:
> > 
> > 1. A different bookmarklet (https://holistica11y.com/text-spacing-bookmarklet-for-accessibility-testing/): <img alt="Screen Shot 2022-08-05 at 11 59 26 AM" width="1044" src="https://user-images.githubusercontent.com/12532727/183115982-d0d77d40-8c5c-44bf-81b7-57741399c397.png">
> > 2. Using the browser's dev tools: <img alt="Screen Shot 2022-08-05 at 2 16 16 PM" width="1044" src="https://user-images.githubusercontent.com/12532727/183137268-f7bb453d-db32-46a7-880c-f6da6a4a1376.png">
> > 
> > Can you send me a link to the bookmarklet you're using or even paste it in a code snippet here?
> 
> Bump @Gopi916 🙂

Hello again, @Gopi916! I am going to close this ticket since I have not been able to reproduce it. Feel free to reopen it if you are still experiencing the issue and can provide steps to reproduce. Thanks!
